portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1144,6 +1147,7 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, geneset_ids,
 		if (data_by_id[geneset_id][case_id].profile_data === null) {
 		    // set the initial value for this sample or patient
 		    data_by_id[geneset_id][case_id].profile_data = sample_score;
+		    data_by_id[geneset_id][case_id].na = false;
```

Comment:
Oh, https://jsperf.com/delete-vs-undefined-vs-null/3 does show that deleting properties takes nearly twice as long as assigning in the browsers I have access to right now, but considering that might be premature optimisation.